### ISSUE TITLE
New submissions for Mon, 21 Feb 22

### ISSUE THREAD

**Issue by LeeKyungwook** [LeeKyungwook/get-arxiv-noti#224]
### Developing Imperceptible Adversarial Patches to Camouflage Military  Assets From Computer Vision Enabled Technologies
 - **Authors:** Authors: Christopher Wise, Jo Plested
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2202.08892
 - **Pdf link:** https://arxiv.org/pdf/2202.08892
 - **Abstract**
 Convolutional neural networks (CNNs) have demonstrated rapid progress and a high level of success in object detection. However, recent evidence has highlighted their vulnerability to adversarial attacks. These attacks are calculated image perturbations or adversarial patches that result in object misclassification or detection suppression. Traditional camouflage methods are impractical when applied to disguise aircraft and other large mobile assets from autonomous detection in intelligence, surveillance and reconnaissance technologies and fifth generation missiles. In this paper we present a unique method that produces imperceptible patches capable of camouflaging large military assets from computer vision-enabled technologies. We developed these patches by maximising object detection loss whilst limiting the patch's colour perceptibility. This work also aims to further the understanding of adversarial examples and their effects on object detection algorithms.
### Classification of ADHD Patients by Kernel Hierarchical Extreme Learning  Machine
 - **Authors:** Authors: Sartaj Ahmed Salman, Zhichao Lian, Yuduo Zhang
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2202.08953
 - **Pdf link:** https://arxiv.org/pdf/2202.08953
 - **Abstract**
 These days, the diagnosis of neuropsychiatric diseases through brain imaging technology has received more and more attention. The exploration of interactions in brain functional connectivity based on functional magnetic resonance imaging (fMRI) data is critical for the study of mental illness. Because attention-deficit/hyperactivity disorder (ADHD) is a chronic disease that affects millions of children, it is difficult to diagnose, so there is still much space for improvement in the accuracy of the diagnosis of the disease. In this paper, we consider the dynamics of brain functional connectivity, modeling a functional brain dynamics model from medical imaging, which helps to find differences in brain function interactions between normal control (NC) children and ADHD children. In more detail, our method is used by Bayesian Connectivity Change Point Model for dynamic detection, Local Binary Encoding Method for local feature extraction, and Kernel Hierarchical Extreme Learning Machine implementation classification. To validate our approach, experimental comparisons of fMRI imaging data on 23 ADHD and 45 NC children were performed, and our experimental methods achieved better classification results than existing methods.
### Out of Distribution Data Detection Using Dropout Bayesian Neural  Networks
 - **Authors:** Authors: Andre T. Nguyen, Fred Lu, Gary Lopez Munoz, Edward Raff, Charles Nicholas, James Holt
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2202.08985
 - **Pdf link:** https://arxiv.org/pdf/2202.08985
 - **Abstract**
 We explore the utility of information contained within a dropout based Bayesian neural network (BNN) for the task of detecting out of distribution (OOD) data. We first show how previous attempts to leverage the randomized embeddings induced by the intermediate layers of a dropout BNN can fail due to the distance metric used. We introduce an alternative approach to measuring embedding uncertainty, justify its use theoretically, and demonstrate how incorporating embedding uncertainty improves OOD data identification across three tasks: image classification, language classification, and malware detection.
### An Active and Contrastive Learning Framework for Fine-Grained Off-Road  Semantic Segmentation
 - **Authors:** Authors: Biao Gao, Xijun Zhao, Huijing Zhao
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Robotics (cs.RO)
 - **Arxiv link:** https://arxiv.org/abs/2202.09002
 - **Pdf link:** https://arxiv.org/pdf/2202.09002
 - **Abstract**
 Off-road semantic segmentation with fine-grained labels is necessary for autonomous vehicles to understand driving scenes, as the coarse-grained road detection can not satisfy off-road vehicles with various mechanical properties. Fine-grained semantic segmentation in off-road scenes usually has no unified category definition due to ambiguous nature environments, and the cost of pixel-wise labeling is extremely high. Furthermore, semantic properties of off-road scenes can be very changeable due to various precipitations, temperature, defoliation, etc. To address these challenges, this research proposes an active and contrastive learning-based method that does not rely on pixel-wise labels, but only on patch-based weak annotations for model learning. There is no need for predefined semantic categories, the contrastive learning-based feature representation and adaptive clustering will discover the category model from scene data. In order to actively adapt to new scenes, a risk evaluation method is proposed to discover and select hard frames with high-risk predictions for supplemental labeling, so as to update the model efficiently. Experiments conducted on our self-developed off-road dataset and DeepScene dataset demonstrate that fine-grained semantic segmentation can be learned with only dozens of weakly labeled frames, and the model can efficiently adapt across scenes by weak supervision, while achieving almost the same level of performance as typical fully supervised baselines.
### Task Specific Attention is one more thing you need for object detection
 - **Authors:** Authors: Sang Yon Lee
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2202.09048
 - **Pdf link:** https://arxiv.org/pdf/2202.09048
 - **Abstract**
 Various models have been proposed to solve the object detection problem. However, most of them require many hand-designed components to demonstrate good performance. To mitigate these issues, Transformer based DETR and its variant Deformable DETR were suggested. They solved much of the complex issue of designing a head of object detection model but it has not been generally clear that the Transformer-based models could be considered as the state-of-the-art method in object detection without doubt. Furthermore, as DETR adapted Transformer method only for the detection head, but still with including CNN for the backbone body, it has not been certain that it would be possible to build the competent end-to-end pipeline with the combination of attention modules. In this paper, we propose that combining several attention modules with our new Task Specific Split Transformer(TSST) is a fairly good enough method to produce the best COCO results without traditionally hand-designed components. By splitting generally purposed attention module into two separated mission specific attention module, the proposed method addresses the way to design simpler object detection models than before. Extensive experiments on the COCO benchmark demonstrate the effectiveness of our approach. Code is released at https://github.com/navervision/tsst
### Guide Local Feature Matching by Overlap Estimation
 - **Authors:** Authors: Ying Chen, Dihe Huang, Shang Xu, Jianlin Liu, Yong Liu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2202.09050
 - **Pdf link:** https://arxiv.org/pdf/2202.09050
 - **Abstract**
 Local image feature matching under large appearance, viewpoint, and distance changes is challenging yet important. Conventional methods detect and match tentative local features across the whole images, with heuristic consistency checks to guarantee reliable matches. In this paper, we introduce a novel Overlap Estimation method conditioned on image pairs with TRansformer, named OETR, to constrain local feature matching in the commonly visible region. OETR performs overlap estimation in a two-step process of feature correlation and then overlap regression. As a preprocessing module, OETR can be plugged into any existing local feature detection and matching pipeline, to mitigate potential view angle or scale variance. Intensive experiments show that OETR can boost state-of-the-art local feature matching performance substantially, especially for image pairs with small shared regions. The code will be publicly available at https://github.com/AbyssGaze/OETR.
### Lightweight Multi-Drone Detection and 3D-Localization via YOLO
 - **Authors:** Authors: Aryan Sharma, Nitik Jain, Mangal Kothari
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2202.09097
 - **Pdf link:** https://arxiv.org/pdf/2202.09097
 - **Abstract**
 In this work, we present and evaluate a method to perform real-time multiple drone detection and three-dimensional localization using state-of-the-art tiny-YOLOv4 object detection algorithm and stereo triangulation. Our computer vision approach eliminates the need for computationally expensive stereo matching algorithms, thereby significantly reducing the memory footprint and making it deployable on embedded systems. Our drone detection system is highly modular (with support for various detection algorithms) and capable of identifying multiple drones in a system, with real-time detection accuracy of up to 77\% with an average FPS of 332 (on Nvidia Titan Xp). We also test the complete pipeline in AirSim environment, detecting drones at a maximum distance of 8 meters, with a mean error of $23\%$ of the distance. We also release the source code for the project, with pre-trained models and the curated synthetic stereo dataset.
### Pinpointing Anomaly Events in Logs from Stability Testing -- N-Grams vs.  Deep-Learning
 - **Authors:** Authors: Mika Mäntylä, Martín Varela, Shayan Hashemi
 - **Subjects:** Software Engineering (cs.SE)
 - **Arxiv link:** https://arxiv.org/abs/2202.09214
 - **Pdf link:** https://arxiv.org/pdf/2202.09214
 - **Abstract**
 As stability testing execution logs can be very long, software engineers need help in locating anomalous events. We develop and evaluate two models for scoring individual log-events for anomalousness, namely an N-Gram model and a Deep Learning model with LSTM (Long short-term memory). Both are trained on normal log sequences only. We evaluate the models with long log sequences of Android stability testing in our company case and with short log sequences from HDFS (Hadoop Distributed File System) public dataset. We evaluate next event prediction accuracy and computational efficiency. The N-Gram model is more accurate in stability testing logs (0.848 vs 0.831), whereas almost identical accuracy is seen in HDFS logs (0.849 vs 0.847). The N-Gram model has superior computational efficiency compared to the Deep model (4 to 13 seconds vs 16 minutes to nearly 4 hours), making it the preferred choice for our case company. Scoring individual log events for anomalousness seems like a good aid for root cause analysis of failing test cases, and our case company plans to add it to its online services. Despite the recent surge in using deep learning in software system anomaly detection, we found no benefits in doing so. However, future work should consider whether our finding holds with different LSTM-model hyper-parameters, other datasets, and with other deep-learning approaches that promise better accuracy and computational efficiency than LSTM based models.
### Assessment of Cyber-Physical Intrusion Detection and Classification for  Industrial Control Systems
 - **Authors:** Authors: Nils Müller, Charalampos Ziras, Kai Heussen
 - **Subjects:** Cryptography and Security (cs.CR); Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2202.09352
 - **Pdf link:** https://arxiv.org/pdf/2202.09352
 - **Abstract**
 The increasing interaction of industrial control systems (ICSs) with public networks and digital devices introduces new cyber threats to power systems and other critical infrastructure. Recent cyber-physical attacks such as Stuxnet and Irongate revealed unexpected ICS vulnerabilities and a need for improved security measures. Intrusion detection systems constitute a key security technology, which typically monitor network data for detecting malicious activities. However, a central characteristic of modern ICSs is the increasing interdependency of physical and cyber network processes. Thus, the integration of network and physical process data is seen as a promising approach to improve predictability in intrusion detection for ICSs by accounting for physical constraints and underlying process patterns. This work systematically assesses real-time cyber-physical intrusion detection and multiclass classification, based on a comparison to its purely network data-based counterpart and evaluation of misclassifications and detection delay. Multiple supervised machine learning models are applied on a recent cyber-physical dataset, describing various cyber attacks and physical faults on a generic ICS. A key finding is that integration of physical process data improves detection and classification of all attack types. In addition, it enables simultaneous processing of attacks and faults, paving the way for holistic cross-domain cause analysis.
## Keyword: face recognition
There is no result 
### Graph Data Augmentation for Graph Machine Learning: A Survey
 - **Authors:** Authors: Tong Zhao, Gang Liu, Stephan Günnemann, Meng Jiang
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2202.08871
 - **Pdf link:** https://arxiv.org/pdf/2202.08871
 - **Abstract**
 Data augmentation has recently seen increased interest in graph machine learning given its ability of creating extra training data and improving model generalization. Despite this recent upsurge, this area is still relatively underexplored, due to the challenges brought by complex, non-Euclidean structure of graph data, which limits the direct analogizing of traditional augmentation operations on other types of data. In this paper, we present a comprehensive and systematic survey of graph data augmentation that summarizes the literature in a structured manner. We first categorize graph data augmentation operations based on the components of graph data they modify or create. Next, we introduce recent advances in graph data augmentation, separating by their learning objectives and methodologies. We conclude by outlining currently unsolved challenges as well as directions for future research. Overall, this paper aims to clarify the landscape of existing literature in graph data augmentation and motivate additional work in this area. We provide a GitHub repository (https://github.com/zhao-tong/graph-data-augmentation-papers) with a reading list that will be continuously updated.
### Dynamic Service Provisioning in the Edge-cloud Continuum with Provable  Guarantees
 - **Authors:** Authors: Itamar Cohen, Carla Fabiana Chiasserini, Paolo Giaccone, Gabriel Scalosub
 - **Subjects:** Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2202.08903
 - **Pdf link:** https://arxiv.org/pdf/2202.08903
 - **Abstract**
 We consider a hierarchical edge-cloud architecture in which services are provided to mobile users as chains of virtual network functions. Each service has specific computation requirements and target delay performance, which require placing the corresponding chain properly and allocating a suitable amount of computing resources. Furthermore, chain migration may be necessary to meet the services' target delay, or convenient to keep the service provisioning cost low. We tackle such issues by formalizing the problem of optimal chain placement and resource allocation in the edge-cloud continuum, taking into account migration, bandwidth, and computation costs. Specifically, we first envision an algorithm that, leveraging resource augmentation, addresses the above problem and provides an upper bound to the amount of resources required to find a feasible solution. We use this algorithm as a building block to devise an efficient approach targeting the minimum-cost solution, while minimizing the required resource augmentation. Our results, obtained through trace-driven, large-scale simulations, show that our solution can provide a feasible solution by using half the amount of resources required by state-of-the-art alternatives.
### Multimodal Emotion Recognition using Transfer Learning from Speaker  Recognition and BERT-based models
 - **Authors:** Authors: Sarala Padi, Seyed Omid Sadjadi, Dinesh Manocha, Ram D. Sriram
 - **Subjects:** Sound (cs.SD); Human-Computer Interaction (cs.HC); Machine Learning (cs.LG); Robotics (cs.RO); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2202.08974
 - **Pdf link:** https://arxiv.org/pdf/2202.08974
 - **Abstract**
 Automatic emotion recognition plays a key role in computer-human interaction as it has the potential to enrich the next-generation artificial intelligence with emotional intelligence. It finds applications in customer and/or representative behavior analysis in call centers, gaming, personal assistants, and social robots, to mention a few. Therefore, there has been an increasing demand to develop robust automatic methods to analyze and recognize the various emotions. In this paper, we propose a neural network-based emotion recognition framework that uses a late fusion of transfer-learned and fine-tuned models from speech and text modalities. More specifically, we i) adapt a residual network (ResNet) based model trained on a large-scale speaker recognition task using transfer learning along with a spectrogram augmentation approach to recognize emotions from speech, and ii) use a fine-tuned bidirectional encoder representations from transformers (BERT) based model to represent and recognize emotions from the text. The proposed system then combines the ResNet and BERT-based model scores using a late fusion strategy to further improve the emotion recognition performance. The proposed multimodal solution addresses the data scarcity limitation in emotion recognition using transfer learning, data augmentation, and fine-tuning, thereby improving the generalization performance of the emotion recognition models. We evaluate the effectiveness of our proposed multimodal approach on the interactive emotional dyadic motion capture (IEMOCAP) dataset. Experimental results indicate that both audio and text-based models improve the emotion recognition performance and that the proposed multimodal solution achieves state-of-the-art results on the IEMOCAP benchmark.
### Prepare your video for streaming with Segue
 - **Authors:** Authors: Melissa Licciardello, Lukas Humbel, Fabian Rohr, Maximilian Grüner, Ankit Singla
 - **Subjects:** Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2202.09112
 - **Pdf link:** https://arxiv.org/pdf/2202.09112
 - **Abstract**
 We identify new opportunities in video streaming, involving the joint consideration of offline video chunking and online rate adaptation. We observe that due to a video's complexity varying over time, certain parts are more likely to cause performance impairments during playback with a particular rate adaptation algorithm. To address this, we propose careful use of variable-length video segments, and augmentation of certain segments with additional bitrate tracks. The key novelty of Segue is in making these decisions based on the video's time varying complexity and the expected rate adaptation behavior over time. We propose and implement several methods for such adaptation-aware chunking. Our results show that Segue substantially reduces rebuffering and quality fluctuations, while maintaining video quality delivered; Segue improves QoE by 9% on average, and by 22% in low-bandwidth conditions. Beyond our specific approach, we view our problem framing as a first step in a new thread on algorithmic and design innovation in video streaming, and leave the reader with several interesting open questions.
### Quantifying the Effects of Data Augmentation
 - **Authors:** Authors: Kevin H. Huang, Peter Orbanz, Morgane Austern
 - **Subjects:** Machine Learning (cs.LG); Statistics Theory (math.ST); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2202.09134
 - **Pdf link:** https://arxiv.org/pdf/2202.09134
 - **Abstract**
 We provide results that exactly quantify how data augmentation affects the convergence rate and variance of estimates. They lead to some unexpected findings: Contrary to common intuition, data augmentation may increase rather than decrease uncertainty of estimates, such as the empirical prediction risk. Our main theoretical tool is a limit theorem for functions of randomly transformed, high-dimensional random vectors. The proof draws on work in probability on noise stability of functions of many variables. The pathological behavior we identify is not a consequence of complex models, but can occur even in the simplest settings -- one of our examples is a linear ridge regressor with two parameters. On the other hand, our results also show that data augmentation can have real, quantifiable benefits.
